src/PublishCodeCovCoverageTask/utils/webUtils.ts

```
@@ -10,208 +10,252 @@ import * as url from 'node:url';
-                            signal, // Pass along the abort signal to the redirected request
-                            onProgress // Pass along the progress callback
-                        }
-                    )
-                    .then(resolve)
-                    .catch(reject);
-                });
-                return;
-            }
+                    // Get total size from headers
+                    const contentLengthHeader = response.headers['content-length'];
+                    const totalBytes = contentLengthHeader !== undefined ? Number.parseInt(contentLengthHeader, 10) : NaN;
```

Biome (1.9.4)

[error] Use Number.NaN instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.NaN instead.


(lint/style/useNumberNamespace)